.travis_deploy.sh

```diff
@@ -3,13 +3,29 @@ set -x
 
 # upload to our artifactory repo for public rpms
 
-REPO="public-rpm"
-
-if [[ "$VERSION" == "el6x" ]]; then
-    REPO="$REPO-el6"
-    PATTERN="*el6*.rpm"
-else
-    PATTERN="*el7*.rpm"
+if [[ -z "${DISTRO_VERS}" ]]; then
+    echo "ERROR: DISTRO_VERS must be set to your minor OS version (e.g. 6.9, 7.4.1708)"
+    exit 1
 fi
 
-./jfrog rt upload "upload_rpms/$PATTERN" $REPO
+case "${DISTRO_VERS}" in
+    "7.x")
```

Comment:
FWIW: if this becomes fragile, I think we could use the centos-release info to find the right location:
```
[root@dino1 pytest-libarchive-nic]# cat /etc/centos-release
CentOS Linux release 7.3.1611 (Core)
```